albumentations/augmentations/transforms.py

```diff
@@ -2471,12 +2473,24 @@ class GaussianBlur(Blur):
         uint8, float32
     """
 
-    def __init__(self, blur_limit=7, always_apply=False, p=0.5):
-        super(GaussianBlur, self).__init__(blur_limit, always_apply, p)
+    def __init__(self, blur_limit=7, min_max_sigma=None, always_apply=False, p=0.5):
+        super(GaussianBlur, self).__init__(always_apply, p)
+        self.blur_limit = to_tuple(blur_limit, 3)
+        self.min_max_sigma = min_max_sigma
 
-    def apply(self, image, ksize=3, **params):
+    def apply(self, image, ksize=3, sigma=0, **params):
         return F.gaussian_blur(image, ksize)
```

Comment:
It is because you do not pass sigma value into `gaussian_blur`

Reply:
Please fix this and add simple test, to check that it is really works.

Reply:
well spotted, I messed it up here
I will have a look on your tests and write some when I find some time
Although I like the idea of leaving opencv to decide on the kernel size, it doesn't help if it uses a size 1 :/
So I'd leave the formula as it is (or use any other). I hope you agree with that.

Reply:
> it doesn't help if it uses a size 1

I do not understand this.
Can you explain this in more detail.